app.ts

```diff
@@ -106,7 +107,8 @@ if (opts.debug) logger.level = 'debug';
 // AP: Detect if we're running under Windows Subsystem for Linux. Temporary modification
 // of process.env is allowed: https://nodejs.org/api/process.html#process_process_env
 if (process.platform === 'linux' && child_process.execSync('uname -a').toString().includes('Microsoft')) {
-    process.env.wsl = true;
+    // Node wants process.env is essentially a Record<key, string | undefined>. Any non-empty string should be fine.
+    process.env.wsl = 'foobar';
```

Comment:
Can maybe do something more "common" like "1"

Reply:
Sounds good to me :)